src/Bicep.Core.IntegrationTests/OutputsTests.cs

```diff
@@ -67,28 +72,31 @@ public void Output_can_have_inferred_resource_type()
         public void Output_can_have_specified_resource_type()
         {
             var result = CompilationHelper.Compile(ServicesWithResourceTyped, @"
-resource resource 'Microsoft.Storage/storageAccounts@2019-06-01' = {
-  name: 'test'
-  location: 'eastus'
-  kind: 'StorageV2'
-  sku: {
-    name: 'Standard_LRS'
-  }
-  identity: {
-    type: 'SystemAssigned'
-  }
-  properties:{
-    accessTier: 'Cool'
-  }
-}
-output out resource 'Microsoft.Storage/storageAccounts@2019-06-01' = resource
-");
+                resource resource 'Microsoft.Storage/storageAccounts@2019-06-01' = {
```

Comment:
[nit] Not super relevant for these tests, but I prefer to keep the correct indenting of the Bicep file, rather than in-lining it with the C#.

Reply:
BTW, I think we can also use this as a chance to replace the verbatim strings in this file with [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/raw-string-literal).

Reply:
Does this look acceptable?
![image](https://github.com/Azure/bicep/assets/9611108/ed581680-ab97-4bed-b103-598723871fec)